java/Oleksii-Riabov/src/com/zoolatech/lecture3/tasks/_3/Shape.java

```diff
@@ -0,0 +1,7 @@
+package com.zoolatech.lecture3.tasks._3;
+
+public sealed interface Shape permits Circle, Triangle, Rectangle {
+    public abstract void findPerimeter();
```

Comment:
all interface methods are public and abstract by default, you don't need to explicitly define them in such way

Reply:
Already remove this redundancy.